wrappers/swift/LogConfiguration.swift

```diff
@@ -0,0 +1,214 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Class wrapping `ODWLogConfiguration` ObjC class object, representing configuration related to events.
+public class LogConfiguration {
+    private var odwLogConfiguration: ODWLogConfiguration
+
+    /**
+    Constructs `self` with provided config as `ODWLogConfiguration` object.
+
+    - Parameters:
+        - config: `ODWLogConfiguration` object which would be wrapped to.
+    */
+    public init(config: ODWLogConfiguration) {
+        self.odwLogConfiguration = config
+    }
+
+    /// Return a copy of the default configuration
+    public static func getLogConfigurationCopy() -> LogConfiguration? {
```

Comment:
What are the scenarios where we aren't able get a copy of the configuration? It may be worth throwing an error if we're unable to retrieve a copy; this is the development paradigm that Swift is moving towards.

Reply:
I don't foresee any scenarios here where it might fail as the objc is creating new object by passing cpp log configuration. Also, objc is not doing any exception handling on its side.

Reply:
Resolving it. Please reopen if any concern.